pkg/controllers/federate/util.go

```diff
@@ -197,6 +230,7 @@ var (
 		util.ConflictResolutionInternalAnnotation,
 		util.OrphanManagedResourcesInternalAnnotation,
 		common.EnableFollowerSchedulingAnnotation,
+		common.SourcePruningMergePatchAnnotation,
```

Comment:
Is this necessary? Isn't the merge patch annotation only added to the federated object?